### ISSUE TITLE
.NET 4.8 WPF TextBox: High Contrast Issues

### ISSUE THREAD

**Issue by rockerinthelocker** [dotnet/wpf#4426]
If a TextBox control receives focus while in high contrast mode, its content is unreadable due to white and gray rectangles placed inside the TextBox.

Until this bug has been fixed, the only workaround available is to use a custom TextBox template for TextBox controls.

First opened here: https://github.com/microsoft/dotnet/issues/1066

**Comment by SamBent** [comment]
@rockerinthelocker  Here's a picture of a `<TextBox Text="TextBox" Width="120"/>` with focus (and some text selected), taken on Win10(21H1) + HighContrast Black + .NET 6.0.0-preview.5.21226.1.  The text is readable, I can't repro your issue.

![image](https://user-images.githubusercontent.com/15221213/118051134-54117180-b335-11eb-98d0-b26fb2bdb919.png)

**Comment by rockerinthelocker** [comment]
@SamBent, Try `<TextBox Text="TextBox" Height="23" Width="120"/>` to repro the issue with .NET 4.8 and .NET Core 5.

Would be great if this gets fixed in .NET 4.8 as well. Thank you.

**Comment by miloush** [comment]
@rockerinthelocker to clarify, the issue you are reporting is that the content does not fit into the `TextBox`, correct?

![image](https://user-images.githubusercontent.com/10546952/119398198-25997c00-bccf-11eb-8973-76cf30f970de.png)

In that case you get what you asked for - fixed height that is too small. If you want make sure the content fits, then you need to let the text box size itself. Different themes can have different size requirements.)

**Comment by ryalanms** [comment]
Thanks, @rockerinthelocker and @miloush.
